.pre-commit-config.yaml

```diff
@@ -52,14 +52,6 @@ repos:
       - id: trailing-whitespace
         exclude: ^.*\.(snap)
         args: ["--markdown-linebreak-ext=md"]
-  - repo: local
-    hooks:
-      - id: eslint
-        name: eslint
-        entry: bash -c 'cd superset-frontend && npm run eslint -- $(echo "$@" | sed "s|superset-frontend/||g")'
```

Comment:
@mistercrunch this sed business seemed to be causing a lot of errors around missing files and whatnot. Is it a problem to remove it?